Code/Framework/AzToolsFramework/AzToolsFramework/Editor/ActionManagerUtils.cpp

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzToolsFramework/Editor/ActionManagerUtils.h>
+
+#include <AzCore/Settings/SettingsRegistry.h>
+
+namespace AzToolsFramework
+{
+    constexpr const char ActionManagerToggleKey[] = "/O3DE/ActionManager/EnableNewActionManager";
```

Comment:
You should mark this as static as well,
```suggestion
    static constexpr AZStd::string_view ActionManagerToggleKey = "/O3DE/ActionManager/EnableNewActionManager";
```